e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -0,0 +1,571 @@
+        String request = json.getString("request");
+        Assert.assertEquals(request, exportResponse.request().uri().toString());
+        JSONArray errors = json.getJSONArray("error");
+        Assert.assertEquals(0, errors.length());
+
+        JSONArray output = json.getJSONArray("output");
+        JSONObject outputObject = output.getJSONObject(0);
+        String url = outputObject.getString("url");
+        Assert.assertEquals(url, AB2D_API_URL + "Job/" + jobUUid + "/file/S0001.ndjson");
+        String type = outputObject.getString("type");
+        Assert.assertEquals(type, "null");
+
+        HttpResponse<String> downloadResponse = fileDownloadRequest(jobUUid, "S0001.ndjson");
+        Assert.assertEquals(200, downloadResponse.statusCode());
+        String fileContent = downloadResponse.body();
+
+        // Some of the data that is returned will be variable and will change from request to request, so not every
+        // JSON object can be verified
+        final JSONObject fileJson = new JSONObject(fileContent);
+        Assert.assertEquals(8, fileJson.length());
+        Assert.assertEquals("ExplanationOfBenefit", fileJson.getString("resourceType"));
+        Assert.assertEquals(0, fileJson.getInt("precedence"));
+        String carrierString = fileJson.getString("id");
+        Assert.assertTrue(carrierString.startsWith("carrier"));
+        final JSONObject patientJson = fileJson.getJSONObject("patient");
+        String referenceString = patientJson.getString("reference");
+        Assert.assertTrue(referenceString.startsWith("Patient"));
+        final JSONObject typeJson = fileJson.getJSONObject("type");
+        Assert.assertNotNull(typeJson);
+        final JSONArray codingJson = typeJson.getJSONArray("coding");
+        Assert.assertNotNull(codingJson);
+        Assert.assertEquals(4, codingJson.length());
+        final JSONArray identifierJson = fileJson.getJSONArray("identifier");
+        Assert.assertNotNull(identifierJson);
+        Assert.assertEquals(2, identifierJson.length());
+        final JSONArray diagnosisJson = fileJson.getJSONArray("diagnosis");
+        Assert.assertNotNull(diagnosisJson);
+        final JSONArray itemJson = fileJson.getJSONArray("item");
+        Assert.assertNotNull(itemJson);
+
+        Map<String, Object> jobData = getJobWithOutput();
+        Assert.assertEquals(jobUUid, jobData.get("job_uuid"));
+        Assert.assertEquals(JobStatus.SUCCESSFUL.name(), jobData.get("status"));
+        Assert.assertEquals("100%", jobData.get("status_message"));
+        Assert.assertEquals(100, jobData.get("progress"));
+        Assert.assertEquals(null, jobData.get("resource_types"));
+        Assert.assertEquals(null, jobData.get("fhir_resource_type"));
+        Assert.assertEquals(0, jobData.get("contract_id")); // In JDBC SQL, null comes back as 0
+    }
+
+    @Test
+    public void runContractNumberExport() throws IOException, InterruptedException, SQLException, JSONException {
```

Comment:
A lot of code in this test method repeats code from the previous one; let's refactor to extract the common code into a reusable method.